.gitignore

```diff
@@ -70,3 +73,5 @@ Gemfile.lock
 setuptools-*.zip
 
 tmp
+# redis tmp config
+redisdb/ci/*.tmp.conf
```

Comment:
maybe `*.tmp` and `*.tmp.conf`, so that we don't have to do this per integration...